website/client/assets/scss/item.scss

```diff
@@ -27,6 +27,12 @@
   background: $white;
   box-shadow: 0 2px 2px 0 rgba($black, 0.15), 0 1px 4px 0 rgba($black, 0.1);
   border: 1px solid transparent;
+<<<<<<< HEAD
```

Comment:
I know this is a WIP so I'm just pointing out to this in case you missed the conflict